apps/web/playwright/bookings-list.e2e.ts

```diff
@@ -95,6 +95,23 @@ test.describe("Bookings", () => {
       await firstGuest.click();
       await expect(titleAndAttendees.locator('[data-testid="unmark-no-show"]')).toBeVisible();
       await expect(titleAndAttendees.locator('[data-testid="mark-no-show"]')).toBeHidden();
+      await webhookReceiver.waitForRequestCount(1);
+      const [request] = webhookReceiver.requestList;
+      const body = request.body;
+      // remove dynamic properties that differs depending on where you run the tests
+      const dynamic = "[redacted/dynamic]";
+      // @ts-expect-error we are modifying the object
+      body.createdAt = dynamic;
+      expect(body).toMatchObject({
+        triggerEvent: "BOOKING_NO_SHOW_UPDATED",
+        createdAt: "[redacted/dynamic]",
+        payload: {
+          message: "x_marked_as_no_show",
```

Comment:
@zomars Why is this checking the translation key and not the actual translation itself? I noticed this while testing locally as well.

Reply:
Is it more that you were using the `message` field to hold a "code" of some sort so the webhook receiver deciphers the message? If so, feels like that should go into a field other than `message`. What do you think?

Reply:
You're right. In the client we run the t function. Makes sense to send it pre-translated. But also the noshow is a public endpoint. So we don't have the user context to get the locale.

Reply:
Made changes to send the message pre-translated.